js/ad_manager_vast.js

```diff
@@ -93,12 +104,47 @@ OO.Ads.manager(function(_, $) {
         OO.log("Invalid VAST XML for tag name: " + rootTagName);
         return false;
       }
-      // TODO, when 3.0 is supported, update this check.
-      if ($(vastXML.firstChild).attr('version') !== '2.0') { return false; }
+
+      var version = getVastVersion(vastXML);
+      if (!supportsVersion(version)) {
+        return false;
+      }
       return true;
     }, this);
 
     /**
+     * Returns the Vast version of the provided XML.
+     * @private
+     * @method Vast#getVastVersion
+     * @returns {number} the Vast version
+     */
+    var getVastVersion = _.bind(function(vastXML) {
+      return $(vastXML.firstChild).attr('version');
+    }, this);
+
+    /**
+     * Checks to see if this ad manager supports a given Vast version.
+     * @private
+     * @method Vast#supportsVersion
+     * @returns {boolean} true if the version is supported by this ad manager, false otherwise
+     */
+    var supportsVersion = _.bind(function(version) {
+      return _.contains(SUPPORTED_VERSIONS, version);
```

Comment:
Got it, I'll add logic for supported Vast versions to check major versions (and minor if necessary).